fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -4392,11 +4393,21 @@ private void createIcebergTable(Database db, CreateTableStmt stmt) throws DdlExc
     private void createHudiTable(Database db, CreateTableStmt stmt) throws DdlException {
         String tableName = stmt.getTableName();
         List<Column> columns = stmt.getColumns();
-        columns.add(new Column("_hoodie_commit_time", Type.STRING, true));
-        columns.add(new Column("_hoodie_commit_seqno", Type.STRING, true));
-        columns.add(new Column("_hoodie_record_key", Type.STRING, true));
-        columns.add(new Column("_hoodie_partition_path", Type.STRING, true));
-        columns.add(new Column("_hoodie_file_name", Type.STRING, true));
+        if (columns.stream().noneMatch(c -> c.getName().equals(HoodieRecord.COMMIT_TIME_METADATA_FIELD))) {
```

Comment:
set = columns.stream().filter(c -> sets.contains(c.getName())).collect(Collectors.toSet());
sets.stream().foreach { 
  s -> if (!set.contains(s)) {
 columns.add(new Column(s, Type.STRING, true));
}
};

Reply:
change to (all-set SUB include-set)